src/doc/uuid_io.h

```diff
@@ -0,0 +1,23 @@
+// Aseprite Document Library
+// Copyright (c) 2025 Igara Studio S.A.
+//
+// This file is released under the terms of the MIT license.
+// Read LICENSE.txt for more information.
+
+#ifndef DOC_UUID_IO_H_INCLUDED
+#define DOC_UUID_IO_H_INCLUDED
+#pragma once
+
+#include "base/uuid.h"
+
+#include <iostream>
```

Comment:
I think `<iosfwd>` is enough here:
```suggestion
#include <iosfwd>
```